src/main/java/com/faforever/client/chat/ChannelTabController.java

```diff
@@ -108,6 +138,30 @@ void initialize() {
     });
   }
 
+  @PostConstruct
+  void init() {
+    assignColors();
+    channelTabScrollPaneVBox.setMinWidth(preferencesService.getPreferences().getChatPrefs().getChannelTabScrollPaneWidth());
+    channelTabScrollPaneVBox.setPrefWidth(preferencesService.getPreferences().getChatPrefs().getChannelTabScrollPaneWidth());
+  }
```

Comment:
If I had found a listener on the splitpane that updates this setting so that its split position is remembred, I would say: Cool! But now I wonder if this is just unfinished or if it's just useless :P